lib/HLW8012_1.1.1/src/HLW8012.cpp

```diff
@@ -186,24 +186,44 @@ void HLW8012::resetEnergy() {
     _cf_pulse_count_total = 0;
 }
 
-void HLW8012::expectedCurrent(float value) {
+void HLW8012::expectedCurrent(float expected) {
     bool valid = false;
     if (static_cast<int>(_current) == 0) getCurrent(valid);
-    if (valid && static_cast<int>(_current) > 0) _current_multiplier *= (value / _current);
+    if (valid && static_cast<int>(_current) > 0) _current_multiplier *= (expected / _current);
```

Comment:
Will `valid` ever be set to `true`, if `_current` is already greater than 0, when calling this method? If not, then the multiplier won't be set, AFAICS...